e2e/cypress/tests/support/api/on_prem_default_config.json

```diff
@@ -41,9 +41,12 @@
         "CorsDebug": false,
         "AllowCookiesForSubdomains": false,
         "ExtendSessionLengthWithActivity": true,
-        "SessionLengthWebInDays": 30,
-        "SessionLengthMobileInDays": 30,
-        "SessionLengthSSOInDays": 30,
+        "SessionLengthWebInDays": 0,
```

Comment:
why are the legacy parameters preserved?

Reply:
> why are the legacy parameters preserved?

Good question. If we remove them it breaks plugins, which I recently discovered the hard way. In a future release (say v8.0) we can remove the old values.